src/shared/model/QueryParams.ts

```diff
@@ -39,6 +39,8 @@ export interface PersistableQueryParams
   fromURI?: string;
   // This is the client Id of a calling application in Okta (ie iOS app etc)
   appClientId?: string;
+  // used to render server side page after redirection based on client state
+  cmp?: boolean;
```

Comment:
Small comment, but since this variable is representing a boolean this could is isCmp